test/functional/apps/visualize/_histogram_request_start.ts

```diff
@@ -71,5 +71,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         });
       });
     });
+
+    describe('autoBounds are not set for number_range data', function () {
+      it('should use provided value when number of generated buckets is less than histogram:maxBars', async function () {
+        log.debug('Field = machine.ram_range');
+        await PageObjects.visEditor.selectField('machine.ram_range');
+        await PageObjects.common.sleep(1000); // fix this
```

Comment:
Maybe we can retry the assertion in the next line instead of sleeping?

Reply:
Done. This pattern is used throughout the file but I don't need it here.